server/index.js

```diff
@@ -49,6 +49,12 @@ app.get("/", (req, res) => {
 	});});
 app.get("/login", (req, res) => res.render(getLoc("login")));
 
+app.get("/logout", (req, res) => {
```

Comment:
Hi, under the rest spec a get request should not make changes to the server side representation.
Get requests are also vulnerable to CSRF attacks; Please change this to post request, use the fetch API (and implement CSRF token)

Thanks!